content/eaas/_index.md

```diff
@@ -78,4 +82,103 @@ The response from the HTTP request will contain a numeric status code indicating
+
+> `curl https://{subdomain}.qrypt.com/api/v1/quantum-entropy?size={kib_entropy} -H "Authorization: Bearer {qrypt_access_token}"`
+
+#### PowerShell
+
+The following shows an example of how to submit a request for *{kib_entropy}* kibibytes of data using the *{qrypt_access_token}* in PowerShell.
+
+\# Specify entropy token, requested size of entropy, and subdomain
+> `[string] $accesstoken = '{qrypt_access_token}'`\
+> `[int] $kibData = {kib_entropy}`\
+> `[string] $sub = '{subdomain}'`
+
+\# Define the request URL
+> `[string] $url = "https://$sub.qrypt.com/api/v1/quantum-entropy?size=$kibData"`
+
+\# Define and submit the request
+> ``$response =  Invoke-RestMethod -Method Get -Uri $url -UseBasicParsing `-Headers @{ authorization = "Bearer $accesstoken" } ` -ContentType 'application/json' ``
+
+\# Display the entropy bytes
+> `response.random | foreach { [Convert]::FromBase64String($_) }`
+
+#### Python
+
+The following shows an example of how to submit a request for *{kib_entropy}* kibibytes of data using the *{qrypt_access_token}* in Python. 
+
+{{% notice note %}}
+You may need to install the “requests” module before executing this example. For example, use the following command: `python -m pip install requests`
+{{% /notice %}}
+
+```
+import requests
+import base64
+
+# Specify entropy token, requeststed size of entropy, and subdomain
+accesstoken = '{qrypt_access_token}'
+kibData = {kib_entropy}
+sub = '{subdomain}'
+
+# Define the request URL
+url = f'https://{sub}.qrypt.com/api/v1/quantum-entropy'
+
+# Define and submit the request
+headers = { 'Authorization': f'Bearer {accesstoken}' }
+params = { 'size': kibData }
+response = requests.get( url, headers=headers, params=params)
+
+$ Displan the entropy bytes
+for s in response.json()['random']: 
+    for b in base64.decodebytes( s.encode('ascii') ): 
+        print( f'{b})
+```
+
+#### JavaScript
+
+The following shows an example of how to submit a request for *{kib_entropy}* kibibytes of data using the *{qrypt_access_token}* in JavaScript.
+
+```
+"use strict"
+
+// Specfify entropy token, requested size of entropy, and subdomain
+let accesstoken = '{qrypt_access_token}'
+let kibData = {kib_entropy}
+let sub = '{subdomain}'
+
+// Define the request URL
+let url = `https://${sub}.qrypt.com/api/v1/quantum-entropy?size=${kibData}`
+
+// Submit the request and process the response
+fetch( url, {
+              method: 'GET',
+              headers: { 'Accept': 'application/json',
+                         'Authorization': "Bearer " + accesstoken },
+            })
+    .then( response => response.json() )
+    .then( function (json) 
+    {
+      // Display the entropy bytes
+      json.random.forEach( b64 => 
+        [...(atob(b64))].forEach( c => console.log( c.charCodeAt(0))) )
+    })
```

Comment:
Would it be possible to make this look nicer? the indentation seems off.

Reply:
Sure, np